presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowStatsRewrite.java

```diff
@@ -200,6 +210,12 @@ else if (expression instanceof IsNullPredicate) {
             else if (expression instanceof IsNotNullPredicate) {
                 validateShowStatsWhereExpression(((IsNotNullPredicate) expression).getValue(), node);
             }
+            else if (expression instanceof InPredicate) {
+                validateShowStatsWhereExpression(((InPredicate) expression).getValue(), node);
```

Comment:
Why do you check only one side of in predicate? What about valueList?
Value list should contain only literals, right?